feature-detects/ambientlight.js

```diff
@@ -12,5 +12,5 @@
 Detects support for the API that provides information about the ambient light levels, as detected by the device's light detector, in terms of lux units.
 */
 define(['Modernizr', 'hasEvent'], function( Modernizr, hasEvent ) {
-  Modernizr.addTest('devicelight', hasEvent('devicelight', window));
```

Comment:
My Customizr tool fails testing due to this unreferenced test name. I could instead change the file name and leave the test name as it was. Open to either.